Development-Guide/CAN-Filter-Guide/README.adoc

```diff
@@ -2,44 +2,44 @@
+. For example, if one is looking for a message from a system that sends out messages with address 0b0000_1111(0x0F) then the ID will be 0b0000_1111(0x0F) because there is only one type of address that is wanted.
 
-.. For example, If you are looking for a message from a system that sends out messages with addresses 0b0001_0001, 0b0001_0010, and 0b0001_0011 (0x11, 0x12, and 0x13) then you want your ID to look like 0b0001_0000 (0x10) because the last two bits do not matter. 
-You could choose 0x11 or 0x15 for your ID or any number for the last hex digit, and it will work regardless. 
-It is your mask that determines which bits are important to compare.
+. For example, if one is looking for a message from a system that sends out messages with addresses 0b0001_0001, 0b0001_0010, and 0b0001_0011 (0x11, 0x12, and 0x13) then the ID will be 0b0001_0000 (0x10) because the last two bits do not matter. 
+0x11, 0x15, or any number for the last hex digit will create a valid ID and it will work regardless. 
 
-. Determine the Mask
 
-.. The Mask will determine which bits in the address of the message and which bits in the ID are important to compare. It compares the messages by using 1s and 0s. You use 1s for the bits that you want to compare and you use 0s for the bits that do not matter. 
-... For example, 0b0000_0001 only compares the first bit (bit 0) while 0b1111_1111 compares every bit
+== Determine the Mask
 
-.. The Mask should be as tight as possible so that no other messages from other addresses except for the one(s) that you want are filtered in. 
-It should also be made for longevity as more systems are prone to be added. 
-Like the ID, the greater the variety of addresses, the more flexible your Mask should be to be able to accommodate the variety.
-... For example, If you are looking for a message from a system that sends out messages with address 0b0000_1111 (0x0F), we know from above that your ID will be 0b0000_1111 (0x0F), but your mask will be 0b1111_1111(0xFF). 
+. The Mask will determine which bits in the address of the message and which bits in the ID are important to compare. It compares the messages by using 1s and 0s. 1s are for the bits that are to be compared and 0s are for the bits that do not matter. 
+.. For example, 0b0000_0001 only compares the first bit (bit 0) while 0b1111_1111 compares every bit.
+
+. The Mask should be as tight as possible so that no other messages from other addresses are filtered through apart from the ones that are desired. 
+It should also be made for longevity as there is a likelihood that more systems will be added. 
+Like the ID, the greater the variety of addresses wanted, the more flexible the Mask should be to be able to accommodate the variety.
+.. For example, if one is looking for a message from a system that sends out messages with address 0b0000_1111 (0x0F), from above the ID will be 0b0000_1111 (0x0F), but the mask will be 0b1111_1111(0xFF). 
```

Comment:
`from above`? this feels like an abstract idea that hasn't been well explained to the reader.